layers/+lang/haskell/README.org

```diff
@@ -93,10 +99,9 @@ checkout this [[https://github.com/kazu-yamamoto/ghc-mod/wiki#known-issues-relat
 
 *** Stack users
 
-Time to time =ghc-mod= is not available via latest Stackage LTS version. So if
-you have problems with calling =stack install ghc-mod=, try to use =stack
-install ghc-mod --resolver lts-3.1= (the last known LTS version that had
-=ghc-mod=). But even if it doesn't work, don't panic, it's easy to install it
+=ghc-mod= should be available via the latest Stackage LTS version. If not, try to use
+=stack install ghc-mod --resolver lts-4.0= (the current LTS which has =ghc-mod=).
```

Comment:
This should be `--resolver lts-4` to make it future proof. If you don't specify the minor number, then stack installs using the latest `lts-4.x` resolver.

Also `ghc-mod` requires that you have the exactly same `ghc` version as your project. So if you're in a project using an `lts-3.x` resolver which uses `ghc 7.10.2`, then `ghc-mod` won't work if it was installed with `lts-4.0` because that uses `ghc 7.10.3`.